package.json

```diff
@@ -163,6 +163,11 @@
                 "type": "boolean",
                 "description": "Enable logging of the Debug Adapter Protocol.",
                 "default": true
+              },
+              "runDebug": {
```

Comment:
For clarity lets rename this to `startDapServer` (or something better.

`runDebug` is a little ambiguous being in an IDE.

Reply:
Its not really disabling the dap server, we still have a dap server, its just not managed by the extension.

`useExistingServer` or something clever like that.